### DISCUSSION TITLE
"enable browser notifications banner" is shown after clearing local storage

### DISCUSSION THREAD

**Discussion by aethiss** [magicbell/community#70] [Questions] [answered]
Hi guys/lads,
we implemented magic bell on our react app, everything works fine, but we have a strange issue about the content section where is displayed to enable the push notification

![image](https://user-images.githubusercontent.com/1488133/219715675-77e3faa3-7da9-4178-ab8b-0d65e87a4436.png)

If i click and enable, everything works as expected, and i also see the push notification on my browser when a new message income. If i refresh the page also all ok. BUT, if i logOut and logIn again, i see the message again (see video)

[Breadcrumbs-push-notification.webm](https://user-images.githubusercontent.com/1488133/219716072-167c9b13-b546-47a8-9120-ba3330f7bdd9.webm)

Our implementation is pretty basic: 

```
import React, { ReactElement } from "react";
import { useSelector } from "react-redux";
import MagicBell, {
  FloatingNotificationInbox
} from "@magicbell/magicbell-react";

import { getCurrentUserId } from "redux/selectors/authSelector";

const bcBellTheme = {
  [cut] <- theme here
};

const MagicBellWrapper = (): ReactElement => {
  const userId = useSelector(getCurrentUserId);

  return (
    <MagicBell
      apiKey={process.env.NEXT_PUBLIC_MAGICBELL_KEY}
      userExternalId={`${userId}`}
      theme={bcBellTheme}
      images={{
        emptyInboxUrl: "/images/logo-ico-color.svg"
      }}>
      {(props) => (
        <FloatingNotificationInbox
          {...props}
          notificationPreferencesEnabled={false}
          height={450}
        />
      )}
    </MagicBell>
  );
};

export default MagicBellWrapper;

```

**Comment by smeijer** [comment, marked as answer]
Hi @aethiss ,

Is there any chance that you clear localStorage on logout? We currently [store a flag in local storage](https://github.com/magicbell-io/magicbell-js/blob/1cb984cfac485254c286385d8a750bc3c62cfbbb/packages/react/src/components/EnablePushNotificationsBanner/EnablePushNotificationsBanner.tsx#L30-L33) to decide if we should [show the subscription banner or hide it.](https://github.com/magicbell-io/magicbell-js/blob/1cb984cfac485254c286385d8a750bc3c62cfbbb/packages/react/src/components/EnablePushNotificationsBanner/EnablePushNotificationsBanner.tsx#L54)
